### PR TITLE
doc(backup): related custom resources might disappear

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#9530

### CODE REVIEW

**Review by jhkrug — APPROVED** [review]
lgtm

**Review by derekbit — APPROVED** [review]

File: content/docs/1.8.0/important-notes/_index.md [hunk 1]
```diff
@@ -208,6 +210,18 @@ For more information, see [Issue #8184](https://github.com/longhorn/longhorn/iss
...
 Longhorn v1.8.0 and later versions support usage of V2 volumes in Talos Linux clusters. To use V2 volumes, ensure that all nodes meet the V2 Data Engine prerequisites. For more information, see [Talos Linux Support: V2 Data Engine](../advanced-resources/os-distro-specific/talos-linux-support#v2-data-engine).
 
+## Backup
+
+### Related Custom Resources Might Disappear
+
+Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+
+- An empty response from the NFS server due to a brief server downtime.
+- A race condition between related Longhorn backup controllers.
```
Comment (innobead) [inline content/docs/1.8.0/important-notes/_index.md:220]: What does this mean?

Comment (derekbit) [reply content/docs/1.8.0/important-notes/_index.md:220]: The detail of the race condition is in https://github.com/longhorn/longhorn/issues/9530#issuecomment-2378253692

Comment (innobead) [reply content/docs/1.8.0/important-notes/_index.md:220]: Just wondered if there is a better brief to describe the situation to make it understandable. For example, 

```
There is a race condition that could potentially delete the remote backup volume and its corresponding backups when resetting the backup target within a short period.
```

Also, can explain what backup-related custom resources are. For example, backup volume and backup, as the default backup target can only be cleaned up/reset by the operators.

File: content/docs/1.6.4/deploy/important-notes/index.md [hunk 2]
```diff
@@ -223,6 +224,16 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
+### Backup Related Custom Resources Might Disappear
```
Comment (derekbit) [inline content/docs/1.6.4/deploy/important-notes/index.md:227]: After rethinking the title and description, not only the customer resources, we need to emphasize the **backup data in backup target** might be deleted. @mantissahz Can you create a PR for this? Thanks.

**Pull request merged**